crates/torii/core/src/sql.rs

```diff
@@ -144,7 +145,7 @@ impl Sql {
 
         let keys_str = felts_sql_string(&keys);
         let insert_entities = "INSERT INTO entities (id, keys, event_id) VALUES (?, ?, ?) ON \
-                               CONFLICT(id) DO UPDATE SET updated_at=CURRENT_TIMESTAMP, \
+                               CONFLICT(id) DO UPDATE SET created_at=CURRENT_TIMESTAMP, \
```

Comment:
Hmm do we want to modify this?

Reply:
Not sure now, thought initially we would remove indexer timestamps & only keep `executed_at` (so block timestamp) - so here we would have to replace the `created_at` timestamp if an entity is written again, right?